docs/wiki/installation/install-windows.md

```diff
@@ -4,10 +4,20 @@ As of osquery 1.8.2+ the Windows builds are feature-complete but provide a limit
 
 Each osquery tag (release) is published to **chocolatey** for our supported versions: [https://chocolatey.org/packages/osquery/](https://chocolatey.org/packages/osquery/)
 
+## Installing osquery
+
+By default Chocolatey will install the binaries, example packs, example configuration, and an openssl certs bundle to `C:\ProgramData\osquery` and nothing more. You can pass Chocolatey the `--params='/InstallService'` flag or make use of osquery's `--install` flag with `C:\ProgramData\osquery\osqueryd\osqueryd.exe --install` to install a Windows system service for the osqueryd daemon.
+
 ## Running osquery
 
-The default install location is `C:\ProgramData\osquery`. The Chocolatey package will also use the `.\osqueryd.exe`'s `--install` and `--uninstall` optional switches to install an osquery service.
+Out of the box osqueryi is runnable via the Chocolatey installation.  If you'd like to run the daemon, you'll need to install the daemon as a service detailed in the steps above, and then provide the daemon with a config file. The simplest way to get osqueryd up and running is to rename the `C:\ProgramData\osquery\osquery.example.conf` file provided to `osquery.conf`. Once the configuration file is in place, you can start the Windows service with `Start-Service osqueryd` if you're using Powershell, or `sc.exe start osqueryd` if your using cmd.exe.
```

Comment:
> Once the configuration file is in place, you can start the Windows service with `Start-Service osqueryd`

This will actually still fail if the `c:\programdata\osquery\log` directory hasn't been created by the user. It doesn't appear to get created automatically by osqueryd or added by chocolatey.